healthtools/views/search_api.py

```diff
@@ -1,12 +1,30 @@
 from flask import Blueprint, request, jsonify
 
 from healthtools.search import run_query
+from healthtools.search.wit_ai import wit_run_query
 
 
 blueprint = Blueprint('search_api', __name__)
 
 
 @blueprint.route('/search', methods=['GET'], strict_slashes=False)
+def search_index(doc_type=None):
+    query = request.args.get('q')
+    result, doc_type = wit_run_query(query, doc_type)
+
+    # Error with wit_run_query (run_query returns false)
+    if(not result):
+        return jsonify({
+            'result': {'hits': [], 'total': 0},
+            'doc_type': doc_type,
+            'status': 'FAILED',
+            'msg': ''  # TODO: Pass run_query message here.
+        })
+
+    # TODO: Log event here (send to Google Analytics)
```

Comment:
This one too

Reply:
This is me again. Notes in code to get back to. Showing where we should do stuff.